### PR TITLE
POL-700 Clean up post-migration dead code

### PR DESCRIPTION
**Pull request by gwenneg**
image

Reviews are OK though 😄

### CODE REVIEW

File: src/main/java/com/redhat/cloud/policies/engine/process/NotificationSender.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ private static String serializeAction(PoliciesAction policiesAction) {
                 .withBundle(BUNDLE_NAME)
                 .withApplication(APP_NAME)
                 .withEventType(EVENT_TYPE_NAME)
-                .withAccountId(policiesAction.getAccountId())
+                .withAccountId("") // TODO Make the accountId optional in the schema.
```
Comment (gwenneg) [inline src/main/java/com/redhat/cloud/policies/engine/process/NotificationSender.java:70]: This TODO must be addressed before the merge.

**Comment by gwenneg** [comment]
@josejulio This PR is ready for a review and a merge :white_check_mark:

**Comment by gwenneg** [comment]
Maybe we should wait for the official end of the Org ID migration on prod before merging :thinking:

File: src/main/java/com/redhat/cloud/policies/engine/db/repositories/PoliciesRepository.java [hunk 2]
```diff
@@ -81,29 +54,16 @@ LocalDateTime findLatestUpdateOrgId(String orgId) {
...
     }
 
-    @CacheInvalidateAll(cacheName = ACCOUNT_LATEST_UPDATE_CACHE_NAME)
-    public void clearAllCaches() {
-        Log.debug("Clearing all caches");
-        enabledPoliciesCache.clear();
-    }
-
     @CacheInvalidateAll(cacheName = ORG_ID_LATEST_UPDATE_CACHE_NAME)
     public void clearAllCachesOrgId() {
```
Comment (josejulio) [inline src/main/java/com/redhat/cloud/policies/engine/db/repositories/PoliciesRepository.java:65]: nitpick: I guess this one should be `clearAllCaches` (for consistency)

**Review by josejulio — APPROVED** [review]
LGTM - one nitpick left in the comments

**Pull request merged**